# jrh_model_choose

Header header



uint32 target_num_choose
uint8 model_choose
